addons/website_portal_sale/models/sale_order.py

```diff
@@ -56,6 +75,34 @@ def _notification_recipients(self, message, groups):
 
         return groups
 
+    @api.model_cr_context
+    def _init_column(self, column_name):
+        """ Initialize the value of the given column for existing rows.
+
+            Overridden here because we skip generating unique access tokens
+            for potentially tons of existing sales orders that won't need it.
+            We do, however, generate it for draft or quotation sales orders.
```

Comment:
On the contrary, I think we do need to generate it for all existing sales orders, for security reasons. Please make sure it is the case.